packages/helpers/src/components/Portal.js

```diff
@@ -11,11 +11,16 @@ const propTypes = {
 const defaultProps = {
   children: null,
   active: true,
-  container: document.body,
+  container: null,
 };
 
-const Portal = ({ children, active, container }) =>
-  active ? ReactDOM.createPortal(children, container) : <React.Fragment>{children}</React.Fragment>;
+function Portal({ children, active, container }) {
```

Comment:
please accept prop parameter and destructure the values inside of the function body, lets be literal that we are accepting those values from the props.